src/app/containers/StoryPromo/index.jsx

```diff
@@ -77,21 +77,30 @@ StoryPromoImage.defaultProps = {
   }),
 };
 
-const LiveComponent = ({ headline, service, dir }) => (
-  // eslint-disable-next-line jsx-a11y/aria-role
-  <span role="text">
-    <LiveLabel service={service} dir={dir}>
-      LIVE
-    </LiveLabel>
-    <VisuallyHiddenText lang="en-GB">{' Live, '}</VisuallyHiddenText>
-    {headline}
-  </span>
-);
+const LiveComponent = ({ headline, service, liveLabel, dir }) => {
+  // As screenreaders mispronounce the word 'LIVE', we use visually hidden
+  // text to read 'Live' instead, which screenreaders pronounce correctly.
```

Comment:
Thanks for adding this explanation. We can reuse this when we pull it out into a psammead component.